content/docs/introduction/roadmap.md

```diff
@@ -11,13 +11,20 @@ features and current work, see the issue trackers for the various repositories,
 for example, the [Prometheus
 server](https://github.com/prometheus/prometheus/issues).
 
+### New storage engine
+
+A new more efficient storage engine is in development. It will offer reduced
+resource usage, better inverted indexes, and generally allow Prometheus to
+scale further.
+
 ### Long-term storage
 
 Currently Prometheus has support for storing samples on local disk, as well as
-experimental support for writing data into OpenTSDB and InfluxDB. We plan to
-improve long-term storage support, although the details are not determined yet.
-This might include read-back support from OpenTSDB/InfluxDB in Prometheus, or
-support for other long-term storage backends.
+experimental support for sending data into remote systems via a generic mechanism.
```

Comment:
As discussed earlier, the term "generic" is confusing here. It is meant as "not specific to the remote system to send to", but it can easily be understood as "sending a generic representation of the sample data" (while it in fact sends the current internal representation, which is subject to change).